src/colorizer/AnnotationData.ts

```diff
@@ -10,18 +10,32 @@ const CSV_COL_ID = "ID";
 const CSV_COL_TIME = "Frame";
 const CSV_COL_TRACK = "Track";
 
+export const BOOLEAN_VALUE = "TRUE";
+
 export const DEFAULT_ANNOTATION_LABEL_COLORS = KNOWN_CATEGORICAL_PALETTES.get(
   DEFAULT_CATEGORICAL_PALETTE_KEY
 )!.colorStops;
 
+export enum LabelType {
+  BOOLEAN = "boolean",
+  INTEGER = "integer",
+  CUSTOM = "custom",
+}
+
 export type LabelOptions = {
+  type: LabelType;
   name: string;
   color: Color;
+  autoIncrement: boolean;
 };
 
 export type LabelData = {
   options: LabelOptions;
+  valueToIds: Map<string, Set<number>>;
   ids: Set<number>;
+  lastValue: string | null;
+  // TODO: Store recently used values? Save values even if all IDs with them
+  // have been removed?
```

Comment:
Will revisit once more UI is implemented! This would be for autocomplete features.